### PR TITLE
AWS: Release v20.1.5

### PR DESCRIPTION
**Pull request by paurosello**
Upgrade Cilium to enhance performance when managing a large number of network policies within the cluster.

### CODE REVIEW

File: aws/v20.1.5/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# :zap: Giant Swarm Release v20.1.5 for AWS :zap:
+
+<< Add description here >>
+
+## Change details
+
```
Comment (Gacko) [inline aws/v20.1.5/README.md:6]:
```suggestion
```

File: aws/v20.1.5/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- Upgrade cilium to v1.15.6.
+- Fix regression setting Policy BPF Max map `policyMapMax` back to 65536 from 16384.
+
+#### Added
+
+Cilium ENI mode for CAPA becomes usable with these changes:
+- Add security group tag filter for pod network
+- Select subnets from secondary VPC CIDRs
+
+
```
Comment (Gacko) [inline aws/v20.1.5/README.md:20]:
```suggestion
```

File: aws/v20.1.5/release.yaml [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+    - vertical-pod-autoscaler-crd
+    name: cert-manager
+    version: 3.7.9
+  - name: chart-operator
+    version: 3.2.1
+  - dependsOn:
+    - prometheus-operator-crd
+    name: chart-operator-extensions
+    version: 1.1.2
+  - dependsOn:
```
Comment (Gacko) [inline aws/v20.1.5/release.yaml:48]: Can we re-add the component version here?

**Review by Gacko — CHANGES_REQUESTED** [review]

File: aws/v20.1.5/README.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# :zap: Giant Swarm Release v20.1.5 for AWS :zap:
+
+<< Add description here >>
+
+## Change details
+
+### cilium [0.25.1](https://github.com/giantswarm/cilium-app/releases/tag/v0.25.1)
+
+#### Changed
```
Comment (Gacko) [inline aws/v20.1.5/README.md:9]:
```suggestion
#### Changed

```

I start feeling bad for these nits. 😬

File: aws/v20.1.5/release.diff [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+    version: 3.2.1                                                   version: 3.2.1
+  - dependsOn:                                                     - dependsOn:
+    - prometheus-operator-crd                                        - prometheus-operator-crd
+    name: chart-operator-extensions                                  name: chart-operator-extensions
+    version: 1.1.2                                                   version: 1.1.2
+  - componentVersion: 1.15.1                                   |   - dependsOn:
+    dependsOn:                                                 < 
+    - aws-cloud-controller-manager                                   - aws-cloud-controller-manager
+    name: cilium                                                     name: cilium
+    version: 0.22.0                                            |     version: 0.25.1
```
Comment (Gacko) [inline aws/v20.1.5/release.diff:52]: Please also re-generate the diff with the component version re-added to the `release.yaml`.

Comment (Gacko) [reply aws/v20.1.5/release.diff:52]: IIRC you can pass the component version to the `devctl` command.

**Review by Gacko — CHANGES_REQUESTED** [review]

**Review by Gacko — APPROVED** [review]

**Review by AndiDog — APPROVED** [review]

**Comment by AndiDog** [comment]
Please describe in the PR summary what this is about (I guess Cilium fixes?)

**Comment by Gacko** [comment]
... and maybe here: https://github.com/giantswarm/releases/pull/1429/files#diff-6056b6b9756343785d02553b9bd6802bad5432c695c90d3a23184642adc80c41R3 😬

**Pull request merged**